CNV/cnv-2.1.0.sh

```diff
@@ -154,3 +154,45 @@ metadata:
   namespace: "${TARGET_NAMESPACE}"
 spec: {}
 EOF
+
+echo "Waiting for HCO to get fully deployed"
+oc wait -n ${TARGET_NAMESPACE} hyperconverged hyperconverged-cluster --for condition=Available --timeout=10m
+
+echo "Setting up a bridge as the default interface"
+while ! default_iface=$(oc get nodenetworkstate -o jsonpath='{.items[0].status.currentState.routes.running[?(@.destination=="0.0.0.0/0")].next-hop-interface}'); do
+    sleep 10
+done
+cat <<EOF | oc apply -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: brext-policy
+spec:
+  desiredState:
+    interfaces:
+    - name: brext
+      type: linux-bridge
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        dhcp: true
+        enabled: true
+      bridge:
+        options:
+          vlan-filtering: true
+          vlans:
+          - vlan-range-min: 2
+            vlan-range-max: 4094
+        port:
+        - name: ${default_interface}
+          vlans:
+          - vlan-range-min: 2
+            vlan-range-max: 4094
+EOF
+
+echo "Waiting until all nodes come back"
+sleep 10
+until oc get nodes 2> /dev/null; do sleep 10; done
```

Comment:
maybe we can add here a check of the hostnetwork CR to check the state contains the new bridge?

Reply:
Good idea

Reply:
Done in the following block. We check that the new machineCIDR route is on the bridge.